packages/main/src/RadioButton.ts

```diff
@@ -0,0 +1,536 @@
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import type I18nBundle from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import type { I18nText } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import ValueState from "@ui5/webcomponents-base/dist/types/ValueState.js";
+import { getEffectiveAriaLabelText } from "@ui5/webcomponents-base/dist/util/AriaLabelHelper.js";
+import type { ClassMap } from "@ui5/webcomponents-base/dist/types.js";
+import {
+	isSpace,
+	isEnter,
+	isDown,
+	isLeft,
+	isUp,
+	isRight,
+} from "@ui5/webcomponents-base/dist/Keys.js";
+import Label from "./Label.js";
+import RadioButtonGroup from "./RadioButtonGroup.js";
+import WrappingType from "./types/WrappingType.js";
+import type { IFormElement } from "./features/InputElementsFormSupport.js";
+import type FormSupport from "./features/InputElementsFormSupport.js";
+
+// Template
+import RadioButtonTemplate from "./generated/templates/RadioButtonTemplate.lit.js";
+
+// i18n
+import {
+	VALUE_STATE_ERROR,
+	VALUE_STATE_WARNING,
+	VALUE_STATE_SUCCESS,
+	VALUE_STATE_INFORMATION,
+	// @ts-ignore
+} from "./generated/i18n/i18n-defaults.js";
+
+// Styles
+import radioButtonCss from "./generated/themes/RadioButton.css.js";
+
+let isGlobalHandlerAttached = false;
+let activeRadio: RadioButton|null = null;
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ * The <code>ui5-radio-button</code> component enables users to select a single option from a set of options.
+ * When a <code>ui5-radio-button</code> is selected by the user, the
+ * <code>change</code> event is fired.
+ * When a <code>ui5-radio-button</code> that is within a group is selected, the one
+ * that was previously selected gets automatically deselected. You can group radio buttons by using the <code>name</code> property.
+ * <br>
+ * <b>Note:</b> If <code>ui5-radio-button</code> is not part of a group, it can be selected once, but can not be deselected back.
+ *
+ * <h3>Keyboard Handling</h3>
+ *
+ * Once the <code>ui5-radio-button</code> is on focus, it might be selected by pressing the Space and Enter keys.
+ * <br>
+ * The Arrow Down/Arrow Up and Arrow Left/Arrow Right keys can be used to change selection between next/previous radio buttons in one group,
+ * while TAB and SHIFT + TAB can be used to enter or leave the radio button group.
+ * <br>
+ * <b>Note:</b> On entering radio button group, the focus goes to the currently selected radio button.
+ *
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import "@ui5/webcomponents/dist/RadioButton";</code>
+ *
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webc.main.RadioButton
+ * @extends sap.ui.webc.base.UI5Element
+ * @tagname ui5-radio-button
+ * @public
+ */
+@customElement("ui5-radio-button")
+@languageAware
+
+/**
+ * Fired when the component checked state changes.
+ *
+ * @event
+ * @public
+ * @since 1.0.0-rc.15
```

Comment:
add @event sap.ui.webc.RadioButton#change otherwise it will not appear in the JSDoc.